helix-core/src/movement.rs

```diff
@@ -149,6 +150,77 @@ fn word_move(slice: RopeSlice, range: Range, count: usize, target: WordMotionTar
     })
 }
 
+pub fn move_prev_para(slice: RopeSlice, range: Range, count: usize, behavior: Movement) -> Range {
```

Comment:
(Snooping around on PRs to get a feel for the code base.)

Would this not be better as `move_prev_paragraph` (emphasis on spelling out paragraph) to disambiguate from parameter? It might also be good in this PR to change all reference "parameter" to "argument," but that is more so  a matter of taste and if the character "a" ends up being used (e.g., "]a").

Reply:
Ah, when I wrote this I made it `para` because it have the same length as `word`, but parameter was there after I rebased, I guess I will change it.